src/data/general-cs-quiz.ts

```diff
@@ -1,4 +1,13 @@
 const generalCSQuiz = [
+  {
+    Question: "What is SOLID?",
+    Answer: "SOLID is an acronym for the first five object-oriented design (OOD) principles by Robert C. Martin",
```

Comment:
Please do not use direct quotes from external sources like Digital Ocean. It is always best to write the answers in your own words 👍 

```suggestion
    Answer: "SOLID is an acronym which represents five principles of Object-Oriented design.",
```